JestConfigEditor test failing

When working on #38, I noticed that one test case is currently failing. It fails not only on my machine, but also [on Travis](https://travis-ci.org/stryker-mutator/stryker-jest-runner/jobs/361525669):

        1) Integration JestConfigEditor
            should create a jest configuration for a react project:

            AssertionError: expected { Object (collectCoverageFrom, setupFiles, ...) } to deeply equal { Object (collectCoverageFrom, setupFiles, ...) }
            + expected - actual

                "src/**/*.{js,jsx,mjs}"
                ]
                "moduleFileExtensions": [
                "web.js"
            +    "mjs"
                "js"
                "json"
                "web.jsx"
                "jsx"
                "node"
            -    "mjs"
                ]
                "moduleNameMapper": {
                "^react-native$": "react-native-web"
                }

--- a/test/integration/JestConfigEditorSpec.ts
+++ b/test/integration/JestConfigEditorSpec.ts
@@ -54,12 +54,12 @@ describe('Integration JestConfigEditor', () => {
       },
       moduleFileExtensions: [
         'web.js',
-        'mjs',
         'js',
         'json',
         'web.jsx',
         'jsx',
-        'node'
+        'node',
+        'mjs'
       ],
       rootDir: projectRoot,
       setupTestFrameworkScriptFile: undefined,

